dwave/system/samplers/clique.py

```diff
@@ -163,11 +163,15 @@ def sample(self, bqm, chain_strength=None, **kwargs):
                 :attr:`.largest_clique_size` variables. This BQM is embedded
                 using a clique embedding.
 
-            chain_strength (float, optional):
-                The (relative) chain strength to use in the embedding. By
-                default a chain strength of `1.5sqrt(N)` where `N` is the size
-                of the largest clique, as returned by
-                :attr:`.largest_clique_size`.
+            chain_strength (float/mapping/callable, optional):
+                Magnitude of the quadratic bias (in SPIN-space) applied between
```

Comment:
This is clearer. 
* It would be more accurate and simple to say "Sets the coupling strength between qubits..."
* It would be good to connect the 1st and 2nd statements. If you prefer two separate statements, then maybe the second could be something like, "The coupling strengths are calculated to penalize chains breaks by an energy of double ..."?

Is it that we add a 2*chain_strength penalty for any broken chain or is it added per qubit that is different from the majority. I'm guessing without checking the former, in which case your "breaks" is correct.

Reply:
Those changes sound good to me! I am not certain about this penalty and am having trouble finding it in the actual source code... I've messaged Kelly and will get back to you on this.

Reply:
I'm going to remove the chain break portion of this docstring (and the other ones as well), as I cannot seem to find it happening in the code and no one seems to know either. It is likely just outdated.